playbooks/group_vars/db.yml

```diff
@@ -1,27 +1,27 @@
 ---
 # mirsg.infrastructure.postgresql - setting up the database
 postgresql_database:
-  database_name: "{{ xnat_db.postgres_xnat_database }}"
-  user_name: "{{ xnat_db.postgres_xnat_user }}"
-  user_password: "{{ xnat_db.postgres_xnat_password }}"
+  database_name: "{{ db_server.postgresql_database }}"
+  user_name: "{{ db_server.postgresql_user }}"
+  user_password: "{{ db_server.postgresql_password }}"
 
 # mirsg.infrastructure.postgresql - connections and authentication
 postgresql_connection:
-  host: "{{ xnat_db.host }}"
-  port: "{{ xnat_db.port }}"
-  client_ip: "{{ xnat_web_server.ip }}"
```

Comment:
`xnat_web_server` has been renamed to `web_server` so that these vars can be used for the database component of OMERO deployments.